src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -20,4 +24,65 @@
       end
     end
   end
+
+  describe 'PUT #put_userinfo' do
+    let(:user) { create(:confirmed_user) }
```

Comment:
This `let(:user) { create(:confirmed_user) }` line is the same as line 8. Should it be deleted?